.github/workflows/build-runs-on-image.yaml

```diff
@@ -0,0 +1,27 @@
+name: Build runs-on image
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
I don't feel too strongly about this, but have you considered moving this (and the entire packer setup actually) to internal-ops?

Reply:
I pulled the workflow out to another PR to internal-ops